src/wp-login.php

```diff
@@ -1103,8 +1103,16 @@ function wp_login_viewport_meta() {
 
 		<p id="nav">
 			<a href="<?php echo esc_url( wp_login_url() ); ?>"><?php _e( 'Log in' ); ?></a>
-				<?php echo esc_html( $login_link_separator ); ?>
-			<a href="<?php echo esc_url( wp_lostpassword_url() ); ?>"><?php _e( 'Lost your password?' ); ?></a>
+			<?php
+
+			echo esc_html( $login_link_separator );
```

Comment:
Why `echo esc_html()` instead of `esc_html_e()`?

```suggestion
			esc_html_e( $login_link_separator );
```

Reply:
Oh I see. It's repeated multiple times within the same file. So here, it's for consistency 👍